magicctapipe/io/tests/test_io.py

```diff
@@ -108,7 +109,7 @@ def test_load_train_data_files_off(gamma_stereo):
     events = load_train_data_files(
         str(gamma_stereo[0]), offaxis_min="0.2 deg", offaxis_max="0.5 deg"
     )
-    data = events["LST1_M2"]
+    data = events["LST1_M1"]
```

Comment:
why the change in here? is it related with a different test file?

Reply:
While running in Git the tests, no LST-M2 event could be found. Even if my laptop could find them. But this already happened some weeks ago by comparing my tests, yours and Alessio's ones